src/cascadia/inc/cppwinrt_utils.h

```diff
@@ -57,6 +57,53 @@ private:
     winrt::event_token className::name(Windows::Foundation::TypedEventHandler<sender, args> const& handler) { return eventHandler.add(handler); } \
     void className::name(winrt::event_token const& token) noexcept { eventHandler.remove(token); }
 
+// This is a helper macro for both declaring the signature of an event, and
+// defining the body. Winrt events need a method for adding a callback to the
+// event and removing the callback. This macro will both declare the method
+// signatures and define them both for you, because they don't really vary from
+// event to event.
+// Use this in a classes header if you have a Windows.Foundation.TypedEventHandler
+#define TYPED_EVENT(name, sender, args)                                                                                                     \
+public:                                                                                                                                     \
+    winrt::event_token name(Windows::Foundation::TypedEventHandler<sender, args> const& handler) { return _##name##Handlers.add(handler); } \
+    void name(winrt::event_token const& token) noexcept { _##name##Handlers.remove(token); }                                                \
+                                                                                                                                            \
+private:                                                                                                                                    \
+    winrt::event<Windows::Foundation::TypedEventHandler<sender, args>> _##name##Handlers;
+
+// // Use this macro to quick implement both the getter and setter for a property.
+// // This should only be used for simple types where there's no logic in the
+// // getter/setter beyond just accessing/updating the value.
+// #define GETSET_PROPERTY(type, name)                   \
+// public:                                               \
+//     type name() const { return _##name; }             \
+//     void name(const type& value) { _##name = value; } \
+//                                                       \
+// private:                                              \
+//     type _##name;
+
+// Use this macro to quick implement both the getter and setter for a property.
+// This should only be used for simple types where there's no logic in the
+// getter/setter beyond just accessing/updating the value.
+#define GETSET_PROPERTY(type, name, defaultValue)     \

```

Comment:
consider using `...` instead of `defaultValue`, and then using the `__VA_ARGS__` expandion down on line 94. That way, you can say `GETSET_PROPERTY(type, name)` and it will fill in defaultValue with [nothing]